deployment/docker_utilities/waku/publisher_headless/traffic.py

```diff
@@ -15,13 +15,21 @@
 
 async def check_dns_time(service: str) -> tuple[str, str, str]:
     start_time = time.time()
-    ip_address = socket.gethostbyname(service)
-    elapsed = (time.time() - start_time) * 1000
-    entire_hostname = socket.gethostbyaddr(ip_address)
-    hostname = entire_hostname[0].split('.')[0]
-    node_shard = int(hostname.split('-')[1])
-    logging.info(f'{service} DNS Response took {elapsed} ms. Resolved to {hostname} with shard {node_shard}.')
-    return f'{ip_address}', hostname, f'{node_shard}'
+    try:
+        ip_address = socket.gethostbyname(service)
+        elapsed = (time.time() - start_time) * 1000
+        entire_hostname = socket.gethostbyaddr(ip_address)
+        try:
+            hostname = entire_hostname[0].split('.')[0]
+            node_shard = int(hostname.split('-')[1])
+            logging.info(f'{service} DNS Response took {elapsed} ms. Resolved to {hostname} with shard {node_shard}.')
+            return f'{ip_address}', hostname, f'{node_shard}'
+        except Exception as e:
```

Comment:
Imo, this general `Exception` can be changed to the ones that can happen here:
```suggestion
        except (IndexError, ValueError) as e:
```